Avatar/avatar.css

```diff
@@ -0,0 +1,125 @@
+body {
+    font-family: sans-serif;
+  }
+
+  :root{
+    --primary-color:#1d4ed8;
+    --text-color:#e5e7eb;
+    --secondary-color:#9ca3af;
+    --bg-color:#e5e7eb;
+}
+
+  .home-header {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    background-color: var(--primary-color);
+    color: var(--text-color);
+    border-radius: 10px;
+  }
+  
+  .brand {
+    flex: 1;
+    margin-left: 1rem;
+    font-weight: 400;
+  }
+  
+  .nav-links {
+    display: flex;
+    list-style: none;
+    flex: 1;
+    justify-content: space-around;
+  }
+  
+  .nav-link {
+    font-family: monospace;
+    text-decoration: none;
+    color: var(--text-color);
+    font-size: 18px;
+  }
+
+  .nav-link:hover{
+    font-size: 20px;
+  }
+
+  .docs-main {
+    display: flex;
+    height: 80vh;
+    width: 100%;
+  }
+  
+  .sidebar {
+    flex: 1;
+    background-color: var(--bg-color);
+    margin: 1rem 0rem;
+  }
+  
+  .docs-main-content {
+    flex: 4;
+  }
+
+  .sidebar-nav-links{
+    list-style: none;
+  }
+
+  .sidebar-nav-link{
+    display: flex;
+    flex-direction: column;
+    justify-content: center; 
+    text-decoration: none;
+    font-family: monospace;
+    font-size: 18px;
+    margin-bottom: 2.5rem;
+    margin-right: 3rem;
+    color: black;
+  }
+
+  .sidebar-nav-link:hover{
+    font-size: 20px;
+    font-weight: 900;
+}
+  
+  .avatar {
+    border-radius: 100px;
+    margin: 1rem;
+  }
+  
+  .large-avatar {
+    max-width: 100px;
+  }
+  
+  .small-avatar {
+    max-width: 80px;
+  }
+
+  .avatar-heading{
+    margin-left: 1rem;
+    font-weight: 500;
+  }
+
+  .avatar-info{
+    margin-left: 2rem;
+    font-size: 20px;
+  }
+
+  .iframe-container{
+    text-align: center;
+    margin-top: 1rem;
+    margin-left: 1rem;
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  iframe{
+    max-width: 450px; 
+    height: 223px; 
```

Comment:
use relative units

Reply:
used flex many times try not to repeat same code again make a separate class for flex

Reply:
@sanjay-xdr done! Thank you so much.